src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -158,12 +158,16 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\extern\Microsoft.Xaml.Behaviors\$(TargetFramework)\Dynamo.Microsoft.Xaml.Behaviors.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Practices.Prism">
+      <HintPath>..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
```

Comment:
I thought @pinzart90 removed this.

Reply:
good catch, I think we should reference the new versions Tibi brought in.

Reply:
@sm6srw could you also move over the net48 specific References into a net48 ItemGroup
Example is not needed for net6 and it causes warnings.
We have very few Assembly References for net6 (as opposed to packagereferences) - only Microsoft.Xaml.Behaviors I think

Reply:
Ok, looks like managed to mess up the merge. This is still WIP. @pinzart Will do.